.circleci/config.yml

```diff
@@ -257,6 +257,11 @@ jobs:
       CMAKE_BUILD_TYPE: RelWithDebugInfo
     steps:
       - checkout
+      - run: # This is not the nicest way. Waiting for feedback from Matthias first.
```

Comment:
We can remove this now, since the docker image is updated.

Reply:
removed.